### PR TITLE
Update Plugins and Correct Names

### PR DESCRIPTION
**Pull request by greyaz**
Plugins:
- ThemeRevision 1.1.10
- Wechat Work Notifier 0.4.1

Names:
- ThemeRevision for Kanboard
- TableView
- EmbedAnything

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1924,18 +1924,18 @@
...
         "homepage": "https://github.com/greyaz/ThemeRevision",
         "is_type": "theme",
-        "last_updated": "2022-12-14",
+        "last_updated": "2023-02-13",
         "license": "MIT",
         "readme": "https://raw.githubusercontent.com/greyaz/ThemeRevision/main/README.md",
         "remote_install": true,
-        "title": "ThemeRevision",
-        "version": "1.1.9"
+        "title": "ThemeRevision for Kanboard",
```
Comment (aljawaid) [inline plugins.json:1937]: This will affect users when installing...

**Review by aljawaid — COMMENTED** [review]
The plugin name and the plugin title must be exact for users to update from within the Kanboard interface

File: plugins.json (hunk 1 above)
Comment (greyaz) [reply plugins.json:1937]: > The plugin name and the plugin title must be exact for users to update from within the Kanboard interface

Yes, this is why I update them. The previous names in my plugins and kanboard web site are not the same.

Comment (aljawaid) [reply plugins.json:1937]: ok but line `1937` was always correct

Comment (greyaz) [reply plugins.json:1937]: > ok but line `1937` was always correct

Actually, the name in my `plugin.php` had been "ThemeRevision for Kanboard" from very early...😂   
So the plugin status display bug has existed for quite a long time.

Comment (aljawaid) [reply plugins.json:1937]: OK but from I know, you can't have any `spaces`

Comment (greyaz) [reply plugins.json:1937]: I think It is allowed.
image
image

Comment (aljawaid) [reply plugins.json:1937]: yes it is allowed... I mean for the update feature to work, you cant have spaces in the plugin name

Comment (greyaz) [reply plugins.json:1937]: Thank you for your reminds 👍, but I made a test, seems it works great. Am I right?
企业微信截图_28d2a630-41bb-4449-aa99-92043d3096fe

Comment (aljawaid) [reply plugins.json:1937]: change the version number and then check

Comment (greyaz) [reply plugins.json:1937]: It works
image

Comment (aljawaid) [reply plugins.json:1937]: OK that's great then

Comment (greyaz) [reply plugins.json:1937]: 👌

**Pull request merged**